src/components/ListView.js

```diff
@@ -7,12 +7,42 @@ const ListView = ({ shoppingList, loading, error }) => {
   const [shoppingListEmpty, setShoppingListEmpty] = useState(true);
   const [length, setLength] = useState(0);
   const [value, setValue] = useState('');
+  const [sortedList, setSortedList] = useState([]);
+  //indicator for urgency status of item
+  let purchaseIndex;
 
   const handleChange = (e) => setValue(e.target.value);
 
+  //assign value to purchaseIndex based on these parameters
+  const checkIndex = (lastPurchase, nextPurchase, timesPurchased) => {
+    if (timesPurchased <= 1 || lastPurchase >= 2 * nextPurchase) {
```

Comment:
Cause the lastPurchasedDate is in milliseconds, to check if time has elapsed, we can't compare days and milliseconds. 

1.  I think we need to first convert the `2*nextPurchase`  from days to milliseconds. 
2.  Add the result to the lastpurchasedDate 
3.  Then check if the result is less than date of now

Reply:
Thanks for the feedback! Multiplying by 2 here is not comparing days and milliseconds though, it's just doubling the estimated time.